docs/changelog/106080.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 106080
+summary: Introduce `TimeSeriesRoutingIdFieldMapper` and use it to create TSDB ids
+area: TSDB
+type: enhancement
```

Comment:
I think this should be a non-issue? It doesn't bring new functionality or improvements on its own. It is an internal refactoring to prepare for another change.